documentation/specs/proposed/RAR_caching_onepager.md

```diff
@@ -0,0 +1,54 @@
+## RAR caching
+
+During every build we need to gather the graph of references and pass
```

Comment:
Can you elaborate a bit on what are these references? Is it project references or something else yet?

Reply:
assembly references